src/Html/Builder.php

```diff
@@ -418,7 +419,14 @@ public function addCheckbox(array $attributes = [])
             'printable'      => true,
             'width'          => '10px',
         ], $attributes);
-        $this->collection->push(new Column($attributes));
+        $column = new Column($attributes);
+	    if ($position === true) {
```

Comment:
I think you mixed up the indent from line 423 to 429 a little bit, there is a tabstop involved and the indent is off by 1.